### PR TITLE
Create a test integration

### PR DESCRIPTION
**Pull request by ninous-dev**

### CODE REVIEW

**Comment by ninous-dev** [comment]
Boohoo Happy day !!

Now dev must be merge (it contains the new testsuite)

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
-* If all tests passed then change the "actual_*.pkl" by the new one before a push
+* Command to run the testsuite on linux: 
+$ python -m pytest -v tests/Test*.py
+* Tests will fail as long as the weights giving a higher accuracy are not put in the best_weights folder
+
+# Test integration
+* Thanks to the python application github's action we add a tests integrations before a pull request merge and at each commit push
+
+# About the heartpredictions module
+TODO
```
Comment (Dianevera) [inline README.md:21]: There is a TODO is it for this issue or another?

File: src/compute_save_compare_accuracies.py [hunk 2]
```diff
@@ -6,14 +6,15 @@
```
Comment (Dianevera) [inline src/compute_save_compare_accuracies.py:6]: useless import: pickle and subprocess

File: src/data_cleaning.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+import os
+import math
+import numpy as np
+
```
Comment (Dianevera) [inline src/data_cleaning.py:5]: numpy and math useless

File: tests/Test_DecisionTree.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 import numpy as np
```
Comment (Dianevera) [inline tests/Test_DecisionTree.py:3]: pytest is useless

File: tests/Test_DecisionTreeAccuracy.py [hunk 5]
```diff
@@ -6,12 +6,13 @@
```
Comment (Dianevera) [inline tests/Test_DecisionTreeAccuracy.py:6]: pytest is useless

File: tests/Test_LogisticRegressionAccuracy.py [hunk 6]
```diff
@@ -8,8 +8,8 @@
```
Comment (Dianevera) [inline tests/Test_LogisticRegressionAccuracy.py:8]: pytest and os are useless

File: README.md (hunk 1 above)
Comment (ninous-dev) [reply README.md:21]: Yes it is for the documentation issue (the 27th issue)

**Pull request merged**